unabridged-works-volume-2/original/09-weighty-question.adoc

```diff
@@ -108,13 +108,19 @@ Is it possible that ever there should be such a state here on earth?
 
 [.discourse-part]
 Answer.
+
+[.numbered-group]
+====
+
+[.numbered]
```

Comment:
Ok, so this is another case where the way you marked it up accidentally ends up putting `Answer.` on it's own line now.  It's tough because it's both a DP and a NG, so I definitely get the confusion.  I think here, I would probably just remove the DP on line 109 (which you didn't even put there), and make `Answer 1.` the first `.numbered` item, like this:

```ASCIIDOC
[.numbered-group]
====

[.numbered]
_Answer 1._ When the principle of God, which lies hid in the hearts of men,
```

Reply:
I put these in a numbered group, but on chapter 8 you had me do them as DP I think? I just did it the way you said on both, and didn't know if it even really matters? It pretty much renders them the same?

Reply:
Yeah, I basically go with what seems like the chunk is _more_ like -- if it's more like a big numbered group, I go with NG, if it's more like a bunch of DP, then I go that route. ¯\_(ツ)_/¯